packtools/sps/validation/article_abstract.py

```diff
@@ -275,16 +275,16 @@ def validate_tag_graphic_in_abstract(self, error_level="ERROR"):
         Returns:
             dict: Formatted validation response if the <graphic> tag is missing.
         """
-        if not self.abstract.get("graphic"):
-            return self._format_response(
-                title="graphic",
-                sub_item="graphic",
-                validation_type="exist",
-                is_valid=False,
-                expected="graphic",
-                advice="Provide graphic",
-                error_level=error_level,
-            )
+        graphic = self.abstract.get("graphic")
+        return self._format_response(
+            title="graphic",
+            sub_item="graphic",
+            validation_type="exist",
+            is_valid=bool(graphic),
+            expected="graphic",
+            advice='mark visual abstract with <abstract abstract-type="graphical"><p><fig><graphic xlink:href=VALUE /> and replace VALUE with figure path',
```

Comment:
mark visual abstract with <abstract abstract-type="graphical"><p><fig><graphic xlink:href="VALUE"/> and replace VALUE with image path

@Rossi-Luciano tem que manter as aspas dos atributos

Reply:
OK, @robertatakenaka